recipes/r-aroma.affymetrix/meta.yaml

```diff
@@ -0,0 +1,115 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-aroma.affymetrix
+  # Note that conda versions cannot contain -, so any -'s in the version have
+  # been replaced with _'s.
+  version: "3.0.0"
+
+source:
+  fn: aroma.affymetrix_3.0.0.tar.gz
+  url:
+    - http://cran.r-project.org/src/contrib/aroma.affymetrix_3.0.0.tar.gz
+    - http://cran.r-project.org/src/contrib/Archive/aroma.affymetrix/aroma.affymetrix_3.0.0.tar.gz
```

Comment:
I just used the provided scripts for generating R and Bioconductor packages. How can I add checksums? How can I know which comments are unneeded?